### ISSUE TITLE
Registration message in general channel is bugged

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2172]
Version: 2.0.3-alpha.15
System: MacOs, Linux, Windows

When I try to reproduce it on android devices (mobile@2.0.3-alpha.16 (iOS 343 ?)) it crashes but I will handle this in another issuee.

I see duplicated gray-out message on user that changed username device. Steps to recreate using usernames that I have on screenshot (steps with messages are not necessary but will help you understand what is going on):

1. Create community with owner with username win1312.
2. Instal Quiet on another device and join with user with username win1312.
3. Disregard "Username already taken" popup and in general channel type message "hello" - it will appear as written by user win1312 with "Duplicate" tag.
4. Type "hi" from owner - it will appear as written by win1312, no tags, it's registered username.
5. Restart app of duplicated user in order to see "Username already taken" popup again.
6. Change username to mac1312.
7. Type "hello" from user mac1312.

As you can see on screen bellow our welcome messages are messed up. This is a view from user that changed username perspective. On other users devices there is no grey out welcome message.

<img width="568" alt="Screenshot 2023-12-13 at 16 07 54" src="https://github.com/TryQuiet/quiet/assets/68909168/6f7af741-e077-404d-ab9d-5e00c630f70b">

Short version of reproducing it:
1. Have an owner with username "test"
2. Join with duplicated user "test"
3. Check what username is showing by writing message.
4. Restart the app in order to see an option to change an username.
5. Change username to something else.
You will se gray out message.

**Comment by siepra** [comment]
This message is going to be send on every community metadata saving (`saveCommunityMetadataSaga`, state-manager).
I don't know why would it cause app crash on mobile, but I also wasn't able to reproduce it.

**Comment by siepra** [comment]
The main problem is this part of code is being executed every time user launches the app:
```
    await this.store.load({ fetchEntryTimeout: 15000 })
    logger('Loaded community metadata to memory')
    const meta = this.getCommunityMetadata()
    if (meta) {
      emitter.emit(StorageEvents.COMMUNITY_METADATA_SAVED, meta)
    }
```
Which causes re-sending the message. It's being greyed-out though, as it happens before general channel is loaded

**Comment by siepra** [comment]
I decided to just delete this part of code as it fixes the problem and unless we have updating community metadata feature, hooking up on a community metadata replication event should not case any troubles

**Comment by holmesworcester** [comment]
Looks good! Should we add a test for this to make sure we don't get a regression in the future?

**Comment by kingalg** [comment]
desktop@2.0.1-alpha.0
mobile@2.0.1-alpha.0 | iOS 346

This solution resolved an issue on both desktop and mobile apps.
